### ISSUE TITLE
Deploykf dashboard not working

### ISSUE THREAD

**Issue by enaalsharairi** [deployKF/deployKF#128]
### Checks

- [X] I have searched the [existing issues](https://github.com/deployKF/deployKF).
- [X] This issue is NOT security related. (Please [disclose responsibly](https://github.com/deployKF/deployKF/security))
- [X] This issue is NOT specific to the CLI. (Please open a [CLI issue](https://github.com/deployKF/cli/issues))

### deployKF Version

version.BuildInfo{Version:"v0.1.2", GitCommit:"243937fa8f8d60b1c3cbe7da2cd68d777ec763df", GitTreeState:"clean", GoVersion:"go1.20.4"}

### Kubernetes Distribution

microk8s

### Kubernetes Version

```shell
MicroK8s v1.29.2 revision 6641

Client Version: v1.29.4
Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
Server Version: v1.29.2
```

### Description

After argocd syncing successfully, deploykf pod running, the deploykf dashboard give no page to present. 

kubectl describe central-dashboard 

```
kubeflow-master@kubeflowmaster:~/deploykf$ kubectl logs central-dashboard-7c9f4c9d79-wz6hg -n deploykf-dashboard

> kubeflow-centraldashboard@0.0.2 start /app
> npm run serve


> kubeflow-centraldashboard@0.0.2 serve /app
> node dist/server.js

Initializing Kubernetes configuration
Unable to fetch Application information: 404 page not found

"other" is not a supported platform for Metrics
Using Profiles service at http://kfam-api.deploykf-dashboard.svc.cluster.local:8081/kfam
Server listening on port http://localhost:8082 (in production mode)
```
`kubectl get svc -n deploykf-istio-gateway
`
```
NAME               TYPE           CLUSTER-IP       EXTERNAL-IP    PORT(S)                                         AGE
deploykf-gateway   LoadBalancer   10.152.183.163   10.64.140.44   15021:30708/TCP,8080:30733/TCP,8443:31352/TCP   24h
```

**Comment by thesuperzapper** [comment]
@enaalsharairi what URL are you trying to access the dashboard with?

Are you using `https://deploykf.example.com:8443/` as [described in the local quickstart](https://www.deploykf.org/guides/local-quickstart/#6-try-the-platform)?

(NOTE: you can't use an IP address to access the dashboard)

**Comment by enaalsharairi** [comment]
Thanks @thesuperzapper,
The connection has been successfully established. I am using the LB external IP, but what I encountered is 'This page isn’t working'

**Comment by enaalsharairi** [comment]
Resolved. it was from 
```
issuer: CN=selfsigned-ca-issuer
*  SSL certificate verify result: unable to get local issuer certificate (20), continuing anyway.
```

**Comment by thesuperzapper** [comment]
@enaalsharairi can you briefly explain what the problem was and how you fixed it so that future people can benefit?

**Comment by enaalsharairi** [comment]
The issue I faced was that the deploykf dashboard webpage was not working, even though all related pods, services, and the istio-gateway were running. I am using a microk8s HA node supervised by an Azure virtual image. It was solved by resyncing the app again in the order mentioned in the deploykf document (https://www.deploykf.org/about/introduction/#about-deploykf), and using a proxy and SSH tunnel on my local machine (Ubuntu OS) to open the dashboard.
